pkg/pit/basecamp.go

```diff
@@ -118,6 +118,7 @@ type BaseCampGlobals struct {
 var k8sRunCMD = []string{
 	"/srv/cray/scripts/metal/net-init.sh",
 	"/srv/cray/scripts/common/update_ca_certs.py",
+	"/srv/cray/scripts/metal/fix_chrony_configs.py",
```

Comment:
@jacobsalmela since metal runs their install with `install.sh`, I think this call should go into there. Does this always need to be ran after cloud-init is (re)started?